### PR TITLE
Update surveys styling for your words 

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

File: tuneme/templatetags/surveypolls.py [hunk 1]
```diff
@@ -11,3 +11,8 @@ def get_surveytype():
 @register.assignment_tag
 def get_pollstype():
     return "polls"
+
+
+@register.assignment_tag
+def get_competitiontype():
```
Comment (codiebeulaine) [inline tuneme/templatetags/surveypolls.py]: perhaps better to say get_survey_type

File: tuneme/templatetags/surveypolls.py [hunk 2]
```diff
@@ -11,3 +11,8 @@ def get_surveytype():
 @register.assignment_tag
 def get_pollstype():
     return "polls"
```
Comment (codiebeulaine) [inline tuneme/templatetags/surveypolls.py]: why do we have this?

File: tuneme/templates/new/surveys/molo_survey_page_landing.html [hunk 3]
```diff
@@ -2,15 +2,20 @@
...
 {% else %}
   {% get_pollstype as surveytype %}
 {% endif %}
-<div class="{{surveytype}} {{surveytype}}--feedback-page">
+<div class="{{surveytype}} {{surveytype}}--feedback-page {% if self.your_words_competition %}yourwords--entries{% endif %}">
 {% if not self.thank_you_text %}
   <h1 class="{{surveytype}}__title">{{ self.title }}</h1>
   <h4 class="{{surveytype}}__intro">{{ self.intro }}</h4>
+{% elif self.your_words_competition %}
+  <h1 class="heading heading--yourwords">{% trans 'Thank you!' %}</h1>
```
Comment (codiebeulaine) [inline tuneme/templates/new/surveys/molo_survey_page_landing.html]: why do we have two Thank you texts here?

File: tuneme/templatetags/surveypolls.py (hunk 2 above)
Comment (projectmushroom) [reply tuneme/templatetags/surveypolls.py]: no idea

File: tuneme/templatetags/surveypolls.py (hunk 1 above)
Comment (projectmushroom) [reply tuneme/templatetags/surveypolls.py]: we already have 

@register.assignment_tag
def get_surveytype():
    return "surveys"

in this file

Comment (codiebeulaine) [reply tuneme/templatetags/surveypolls.py]: all of these should be in one template tag, with some logic to return the correct type, we can do it on a call

File: tuneme/templates/new/surveys/molo_survey_page_landing.html (hunk 3 above)
Comment (projectmushroom) [reply tuneme/templates/new/surveys/molo_survey_page_landing.html]: I've only added the Thank you to the your words success page, the default survey does not show it

File: tuneme/templates/new/surveys/molo_survey_page.html [hunk 4]
```diff
@@ -1,35 +1,78 @@
 {% extends 'base.html' %}
-{% load wagtailcore_tags surveypolls i18n %}
+{% load wagtailcore_tags wagtailimages_tags core_tags surveypolls i18n %}
 
 {% block content %}
-{% if not self.display_survey_directly %}
+{% if self.your_words_competition %}
+  {% get_competitiontype as surveytype %}
+{% elif not self.display_survey_directly %}
```
Comment (codiebeulaine) [inline tuneme/templates/new/surveys/molo_survey_page.html]: @projectmushroom can we not just say `'yourwords' as surveytype`

File: tuneme/templates/new/surveys/surveys_list.html [hunk 5]
```diff
@@ -1,24 +1,74 @@
-{% load wagtailcore_tags core_tags surveypolls i18n %}
+{% load wagtailcore_tags core_tags surveypolls wagtailimages_tags i18n %}
 {% block content %}
 {% for survey in surveys %}
 {% with survey=survey.molo_survey_page form=survey.form %}
 
-{% if not survey.display_survey_directly %}
+{% if survey.your_words_competition %}
+  {% get_competitiontype as surveytype %}
```
Comment (codiebeulaine) [inline tuneme/templates/new/surveys/surveys_list.html]: same here with surveytype

File: tuneme/templatetags/surveypolls.py (hunk 2 above)
Comment (codiebeulaine) [reply tuneme/templatetags/surveypolls.py]: these shouldn't exist at all

**Comment by smarzban** [comment]
👍

**Pull request merged**